docs/install_guides/installing-openvino-yum.md

```diff
@@ -1,144 +1,123 @@
 
-> **NOTE**: You must be logged in as root to set up and install the repository.
-<br>
-Configure YUM with the OpenVINO repository to install OpenVINO. You have two options for this, using the `yum-config-manager` or manually by creating a text file and pointing YUM to the file. 
+### Step 1: Set up the repository
 
-* **OPTION 1:** Import the `.repo` file using the `yum-config-manager`:
-   1. `yum-utils` must be installed on your system.  If it’s not currently installed, run the command:
-   ```sh
-   sudo yum install yum-utils
-   ```
-   2. Add repository using the `yum-config-manager`:
-   ```sh
-   sudo yum-config-manager --add-repo https://yum.repos.intel.com/openvino/2021/setup/intel-openvino-2021.repo
-   ```
-   3. Import the gpg public key for the repository:
-   ```sh
-   sudo rpm --import https://yum.repos.intel.com/openvino/2021/setup/RPM-GPG-KEY-INTEL-OPENVINO-2021
+1. Create the YUM repo file in the `/tmp` directory as a normal user:
    ```
-
-* **OPTION 2:** Create the repository file manually:
-
-   1. Create the YUM repo file in the /tmp directory as a normal user:
-   ```sh
-   tee > /tmp/openvino-2021.repo << EOF
-   [intel-openvino-2021]
-   name=Intel(R) Distribution of OpenVINO 2021
-   baseurl=https://yum.repos.intel.com/openvino/2021
+   tee > /tmp/openvino-2022.repo << EOF
+   [OpenVINO]
+   name=Intel(R) Distribution of OpenVINO 2022
+   baseurl=https://yum.repos.intel.com/openvino/2022
    enabled=1
    gpgcheck=1
-   gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-INTEL-OPENVINO-2021
+   repo_gpgcheck=1
+   gpgkey=https://yum.repos.intel.com/intel-gpg-keys/GPG-PUB-KEY-INTEL-SW-PRODUCTS.PUB
    EOF
    ```
-   2. Move the newly created openvino-2021.repo file to the YUM configuration directory /etc/yum.repos.d:
-   ```sh 
-   sudo mv /tmp/openvino-2021.repo /etc/yum.repos.d
+2.	Move the new openvino-2022.repo file to the YUM configuration directory `/etc/yum.repos.d`:
+   ```sh
+   sudo mv /tmp/openvino-2022.repo /etc/yum.repos.d
    ```
-   3. Import the gpg public key for the repository:
+3.	Verify that the new repo is properly setup by running the following command:
    ```sh
-   sudo rpm --import https://yum.repos.intel.com/openvino/2021/setup/RPM-GPG-KEY-INTEL-OPENVINO-2021
+   yum repolist | grep -i openvino
    ```
+    You will see the available list of packages.
 
-### Verify that the new repo is properly setup
-Run the following command:   
-```sh
-yum repolist | grep -i openvino
-```
 
-Results:
-```sh
-intel-openvino-2021 Intel(R) Distribution of OpenVINO 2021
+To list available OpenVINO packages, use the following command:
 ```
-  
-### To list available OpenVINO packages
-Use the following command:
-```sh
-yum list intel-openvino*
+yum list 'openvino*'
 ```
 
----
-  
-## Install Runtime Packages Using the YUM Package Manager
+### Step 2: Install OpenVINO Runtime using the YUM Package Manager
+
+Intel® OpenVINO will be installed in: `/opt/intel/openvino_<VERSION>.<UPDATE>.<PATCH>`
```

Comment:
Same comment as for apt: since 2022 is already in lines 33-35, can <VERSION> be replaced?

Reply:
same as above